### PR TITLE
Ixs 2765 update vetting form

### PR DESCRIPTION
**Pull request by harryng22**

### CODE REVIEW

File: src/components/LaunchpadIssuance/IssuanceForm/Vetting/constants.ts [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import { SMART_CONTRACT_STRATEGIES } from "components/LaunchpadIssuance/types";
+
+export const offeringData = [
```
Comment (witchaotic) [inline src/components/LaunchpadIssuance/IssuanceForm/Vetting/constants.ts:3]: could you please rename it to strategyOptions? offeringData sound like an object containing data about the offer.

**Review by witchaotic — CHANGES_REQUESTED** [review]

File: src/assets/launchpad/svg/info-icon.svg
[the remark this answers is not in the document]
Comment (harryng22) [reply src/assets/launchpad/svg/info-icon.svg:1]: I have resolved it

File: src/components/LaunchpadIssuance/IssuanceForm/Vetting/index.tsx [hunk 2]
```diff
@@ -193,6 +197,23 @@ export const IssuanceVettingForm = ({ view = false }: IssuanceVettingFormProps)
           </FormSideBar>
 
           <FormBody>
+            <StrategyInfoBlock>
+              {strategyOptions.map((offer, idx) => <StrategyCard key={offer.option + idx}
+                field="smartContractStrategy"
+                id={offer.option}
+                checked={offer.option === (values.smartContractStrategy ? values.smartContractStrategy : defaultValues.smartContractStrategy)}
```
Comment (witchaotic) [inline src/components/LaunchpadIssuance/IssuanceForm/Vetting/index.tsx:204]: we have `useVettingFormInitialValues` that can be used for setting default values, so you can use here just `values.smartContractStrategy`

File: src/components/LaunchpadIssuance/IssuanceForm/Vetting/index.tsx [hunk 3]
```diff
@@ -193,6 +197,23 @@ export const IssuanceVettingForm = ({ view = false }: IssuanceVettingFormProps)
           </FormSideBar>
 
           <FormBody>
+            <StrategyInfoBlock>
+              {strategyOptions.map((offer, idx) => <StrategyCard key={offer.option + idx}
```
Comment (witchaotic) [inline src/components/LaunchpadIssuance/IssuanceForm/Vetting/index.tsx:201]: how about you rename `offer` here to strategy/option/item?

**Review by witchaotic — CHANGES_REQUESTED** [review]

File: src/components/LaunchpadIssuance/IssuanceForm/Vetting/index.tsx (hunk 2 above)
Comment (harryng22) [reply src/components/LaunchpadIssuance/IssuanceForm/Vetting/index.tsx:204]: This is radio button, so we need this expression for the checked property

**Pull request merged**